.circleci/config.yml

```diff
@@ -91,9 +91,6 @@ commands:
   create-git-tag:
     steps:
       - run: *gh-config
-      - run:
-          name: Current git tags
-          command: git tag
```

Comment:
Same fix as in the templates repo.  Basically this command hangs due to the number of tags being displayed.  Just remove it.